src/stories/other.stories.tsx

```diff
@@ -77,3 +78,14 @@ stories.add('Circle Icon', withInfo(
         {text('Text', '1')}
     </CircleIcon>
 ));
+
+stories.add('Keep aspect ratio', withInfo(
+    'A container to keep a given aspect ratio for an image or iframe'
+)(() =>
+    <div>
+        <div style={{ width: text('Width', '40rem'), backgroundColor: '#DDD'}} >
+            <KeepAspectRatio aspectRatio={(text('Aspect ratio', 16 / 9))} className='my-picture-frame' content={<iframe className='interesting-video' width="560" height="315" src="https://www.youtube.com/embed/160yAufQ-is" allowFullScreen></iframe>} >
```

Comment:
Can you just use a grey rectangle instead of the youtube video to simplify what this is showing